science/hdf5/Portfile

```diff
@@ -115,6 +115,9 @@ variant cxx description {
 } {
     configure.args-delete       --disable-cxx
     configure.args-append       --enable-cxx
+    if {[gcc_variant_isset]} {
+        configure.cxx_stdlib        libstdc++
```

Comment:
Should be `macports-libstdc++` if it's using the libstdc++ provided by the libgcc port.